src/web/extractors.rs

```diff
@@ -1216,6 +1217,10 @@ impl FromRequest for BsoQueryParams {
         Box::pin(async move {
             let params = Query::<BsoQueryParams>::from_request(&req, &mut payload)
                 .map_err(|e| {
+                    let mut exts = req.extensions_mut();
+                    let mut tags = Tags::default();
+                    tags.add_extra("query_params", req.query_string());
```

Comment:
We already have this in sentry, see `uri.path` [in extras](https://sentry.prod.mozaws.net/operations/syncstorage-prod/issues/10887369/?referrer=github_plugin) which includes the params.

So the `BsoQueryParams` error's happening due to a client passing in an encoded `offset` (with a semicolon) and is probably happening mostly on ios. Is the client attempting to use encoded offsets despite the server not supplying them as encoded?

Reply:
Yeah, that has to be the reason. I see that there's [a comment](https://github.com/mozilla-services/syncstorage-rs/blob/master/src/web/extractors.rs#L1143-L1148) where we don't process the encoded offset . I'm guessing we must still be leaking that form of offset.

Reply:
Yep, we're still sending out the encoded format: https://github.com/mozilla-services/syncstorage-rs/blob/master/src/web/extractors.rs#L1135

Reply:
but AFAICT we never set a timestamp _anywhere_ after we disabled these offsets, so I don't think we're actually emitting them.

Reply:
Well, as far as I understand, we're the only thing that generates the offset, so I'm guessing it's getting leaked somehow.

Reply:
We can certainly remove timestamp and this code path just in case (and wait for it to deploy before looking into ios further) but I don't see it changing anything.

IIRC we've seen this error since going back to last year when we first deployed. I had a theory it was a value "leftover" from the old python server but I don't think that's a possibility: the client resets its state when assigned a new node.

Granted the old sentry event isn't around any longer and we never tracked this with an issue so I'm fine disregarding my recollection of all this, removing the code path and going from there 😄

Reply:
For now, I commented out the timestamp aspect of the Offset. I'm not going to do the full surgery since eventually we'd need it, and that would touch A LOT of code.